### DISCUSSION TITLE
Custom Load Balancing - Specifically target nodes

### DISCUSSION THREAD

**Discussion by marcopasveer** [ravendb/ravendb#13579] [Q&A] [answered]
Hey,

We have a 4 node cluster. 2 nodes on premise (internal location), 2 in our private cloud (external location).
The 4 nodes keep eachother in sync through a VPN spanning the 2 locations.

We use RavenDB as our backend for our website selling a various range of products.
This website runs both internally and externally on the private cloud.

When our RavenDb client makes a connection with a node it retrieves the topology and starts load balancing between all 4 nodes.
Is there a way to read specifically from only 2 nodes instead of all 4? (Preferably round robin between the 2 nodes)
We do not want to read & write from out external location since this takes a longer time to load.

Kind regards, 

Marco Pasveer

**Comment by ayende** [comment, marked as answer]
You can setup the routing to use `FastestNode`, which will use the nearby node without you really having to do much about it.
